e2e/integrations/edit/edit.po.ts

```diff
@@ -57,6 +67,26 @@ export class ConnectionSelectComponent implements IPaaSComponent {
 }
 
 
+export class IntegrationBasicsComponent implements IPaaSComponent {
+  static readonly nameSelector = 'input[name="nameInput"]';
```

Comment:
I wonder if Protractor supports selecting by the model property instead of by the field property. It's worth looking into. I know they did for Angular 1.x but haven't had a chance to check for 2.x. Either way, this is better than using an ID or class. 👍